### ISSUE TITLE
Add Material You theme support for Android

### ISSUE THREAD

**Issue by swmarks** [liftoff-app/liftoff#109]
Right now, the color palette for either of the themes (light/dark) might not be to everyone's liking. Luckily, Google has added theming options to Material UI that grab colors from the system theme with the ability to fallback to default values. I would love to see Material You support added.

**Comment by zachatrocity** [comment]
We need to be cautious here as we're a cross platform app. 

@mykdavies and myself really want to put an emphasis on design and what that would mean for the app on both platforms.

Is that material you? Perhaps. Or it might be some else that makes our app stand out.

Thanks for making this ticket, let's use it as a conversation and ideation space on where we want to take the design of the app.

**Comment by zachatrocity** [comment]
To clarify, I want to do material 3 and I think it would be pretty straightforward.

I just wanna make sure we have an understanding on what (if any) consequences that would have on iOS

**Comment by mykdavies** [comment]
Yeah, I have no idea tbh, sorry. But if it's just another checkbox in the "theme" list we can just hide that choice when not on Android.
